### PR TITLE
Expand details about the collected pane item data

### LINKED ISSUE #57: What does 'The name of each item opened in a pane such as...' mean?
Hi,  I'm new to Atom having only installed it this morning and played around with it for a bit.  So far I'm liking it a lot.  I wasn't too happy to find it had tracking enabled by default but I won't get into that now.

I saw this item in the collected data:

> The name of each item opened in a pane such as EditorView, SettingsView, and MarkdownPreviewView

Which almost sounds like it would collect filenames.  I didn't think that would be likely and I found issue #16 which talked about the tracking and had this comment:

> No data about filenames or file contents is sent.

That's good to here but I'd be interested if anyone can easily explain what names that original point is referring to.  Any help would be appreciated.

I'm a developer and I know that all the code for this is available.  Once I've started learning more about Atom and the packages I should be able to answer this myself.  I've got one problems if someone wants to close this issue until I can do that.

### PR DESCRIPTION
**Pull request by thomasjo**
Not sure the `No other pane item information is collected` part is necessary though. The slight re-phrasing should make it super-clear what information is collected.

---
Resolves #57

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ Metrics package.
 * A unique UUID v4 random identifier is generated according to [RFC4122][RFC4122]
 * The screen width and height
 * The version of Atom being used
-* The name of each item opened in a pane such as `EditorView`, `SettingsView`,
-  and `MarkdownPreviewView`
+* The name of each item opened in a pane, e.g. `EditorView`, `SettingsView`,
```
Comment (lee-dohm) [inline README.md]: I would rephrase this to something more like "The name of each view class opened in a pane, e.g. ..."

Comment (mnquintana) [inline README.md]: > I would rephrase this to something more like "The name of each view class opened in a pane, e.g. ..."

:+1: For clarity there. Other than that I think this looks good :rocket:

Comment (thomasjo) [inline README.md]: > I would rephrase this to something more like "The name of each view class opened in a pane, e.g. ..."

What about *"The name of each view class or atom configuration file opened in a pane, e.g. ..."*, and then making sure to mention for instance the name/key `UserKeymap` in the list of examples?

Comment (lee-dohm) [inline README.md]: :+1: Sounds good to me.
